### DISCUSSION TITLE
Zonal statistics using geopandas GeoDataFrame and/or shapely objects?

### DISCUSSION THREAD

**Discussion by vidurmithal** [makepath/xarray-spatial#656] [Ideas]
Most use-cases of zonal statistics would have the geometries stored in a `geopandas` `GeoDataFrame`, rather than as an `xarray` `DataArray`. Converting a `GeoDataFrame` to an `DataArray` with "zones" is relatively simple using the `regionmask` package.

**Comment by brendancol** [comment]
@vidurmithal sorry for the late response. right now xarray-spatial does not support rasterization and tend to point folks to datashader, gdal and other packages.
